go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/open-quantum-safe/liboqs-go
 
-go 1.11
+go 1.18
```

Comment:
This is an apparent "global" change that @vsoftco should review/approve: I don't know enough about Go to evaluate the impact of this change.

Reply:
will look into it, thanks!

Reply:
But then again, if the (self-)tests run OK, this should be OK, right?